pudb/debugger.py

```diff
@@ -710,6 +712,7 @@ def __init__(self, dbg, stdin, stdout, term_size):
         self.search_controller = SearchController(self)
 
         self.last_module_filter = ""
+        self.var_focus_index = None
```

Comment:
Not a fan of this being global state. As far as I can tell, it's only being used to get the index from `collapse_current` into `update_var_view`. I would much prefer if the lifetime of this state were limited to the subroutine calling both `change_var_state`. If it hangs around longer, it just produces questions regarding its validity.

Reply:
Done! Thanks for pointing that out, I hadn't spotted that there was an easy way to connect these two.